lib/setupmanagers/qemuhck/states.json

```diff
@@ -51,5 +51,12 @@
         "true": {
             "drive_cache_options": ",cache=unsafe"
         }
+    },
+    "fw_type": {
+        "uefi": {
+            "devices_list": [
+                "tpm-tis"
```

Comment:
We need to update the devices_list processing logic in this case.  The current PR has a problem with running SVVP. `tpm-tis` occurs twice in devices_list and auto_hck fails when trying to start the TPM server again.

Reply:
Done with commit e3ee8668b9bdccd10c375f9dfd44c8f4795e1cf1.